example/docker-compose/docker-compose.hotrod.yaml

```diff
@@ -0,0 +1,87 @@
+version: "2"
+services:
+
+  tempo:
+    image: grafana/tempo:latest
+    command: 
+      - "-storage.trace.backend=local"                  # tell tempo where to permanently put traces
+      - "-storage.trace.local.path=/tmp/tempo/traces"   
+      - "-storage.trace.wal.path=/tmp/tempo/wal"        # tell tempo where to store the wal
+      - "-auth.enabled=false"                           # disables the requirement for the X-Scope-OrgID header
+      - "-server.http-listen-port=3100" 
+    volumes:
+      - ./tempo.yaml:/etc/tempo.yaml
+      - /tmp/tempo:/tmp/tempo
```

Comment:
this has changed in the examples.  check the tip of master for the new place we put tempo data files in examples.